examples/hello_world/lib/main.dart

```diff
@@ -2,6 +2,54 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
 import 'package:flutter/widgets.dart';
 
-void main() => runApp(const Center(child: const Text('Hello, world!', textDirection: TextDirection.ltr)));
```

Comment:
Please don't change the hello world example. The whole point of this example is to be a one liner.

Reply:
Sorry commited by mistake, was using this to debug values - will revert

Reply:
Fixed